src/spikeinterface/generation/noise_tools.py

```diff
@@ -7,22 +7,25 @@ def generate_noise(
     probe, sampling_frequency, durations, dtype="float32", noise_levels=15.0, spatial_decay=None, seed=None
 ):
     """
+    Generate a noise recording.
 
     Parameters
     ----------
     probe : Probe
         A probe object.
     sampling_frequency : float
-        Sampling frequency
+        The sampling frequency of the recording.
     durations : list of float
-        Durations
+        The duration(s) of the recording.
```

Comment:
```suggestion
        The duration(s) of the recording(s) in seconds.
```

I'm reading the (s) as an optional pluralisation, depending on whether you're making several recording segments using a list, or a single recording segment using a float? Maybe it's meant to mean seconds?

Reply:
Maybe just:
```suggestion
        The duration(s) of the recording segment(s).
```